packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -269,6 +269,9 @@ class UserMessages {
   String get flutterNoDevelopmentDevice =>
       "Unable to locate a development device; please run 'flutter doctor' "
       'for information about installing additional components.';
+  String flutterSpecifiedDeviceDevModeDisabled(String deviceName) => 'To use '
```

Comment:
I would move this to target_devices.dart and make it private since that's the only place it's needed and used. The purpose of User Messages was originally for g3 overrides.